### PR TITLE
Test implementation

### PR DESCRIPTION
**Pull request by Kevin-Guillen**
movielisttest

apitest

### CODE REVIEW

File: test/api_test.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    () {
+      moviesApiProvider = MoviesApiProvider();
+    },
+  );
+
+  group(
+    "Api testing",
+    () {
+      test(
+        "Trending Movies",
```
Comment (FranciscoGaraventa) [inline test/api_test.dart]: Try to add a more descriptive test name like "Should be fetch trending movies when fetchTrendingMovies is call" or something like that

Comment (Kevin-Guillen) [reply test/api_test.dart]: done

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Pull request merged**